presto-product-tests/README.md

```diff
@@ -93,12 +93,20 @@ where ``<profile>`` is one of either:
 - **multinode** - pseudo-distributed Hadoop installation running on a
  single Docker container and a distributed Presto installation running on
  multiple Docker containers. For multinode the default configuration is 1 coordinator and 1 worker.
-- **singlenode** - pseudo-distributed kerberized Hadoop installation running on a
- single Docker container and a single node installation of kerberized Presto also running
- on a single Docker container.
-- **singlenode-kerberized** - pseudo-distributed Hadoop installation running on a
+- **singlenode** - pseudo-distributed Hadoop installation running on a
  single Docker container and a single node installation of Presto also running
  on a single Docker container.
+- **singlenode-kerberized** - pseudo-distributed kerberized Hadoop installation running on a
```

Comment:
I propose that these should be called singlenode-kerberos-impersonation and singlenode-kerberos-no-impersonation.